src/overview/components/Overview.jsx

```diff
@@ -18,7 +19,7 @@ class Overview extends React.Component {
                 ref='inputQuery'
             >
             </input>
-            <ResultList searchResult={this.props.searchResult} />
+            {this.props.waitingForResults==0 ? <ResultList searchResult={this.props.searchResult} />: <LoadingIndicator />  }
```

Comment:
I tried removing the ==0 but it wasn't working. For now I have kept it the same unless we can add a function something like this
```
waitingForResults: waitingForResults:ourState(state).waitingForResults==0 ? true:false
```

But that is essentially doing the same thing